src/components/content/catalog/services/policies/addPolicy/ServicePolicyCreateResultStatus.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { Alert } from 'antd';
+import React from 'react';
+import ServicePolicySubmitResult from '../ServicePolicySubmitResult';
+import { ApiError, Response, ServicePolicy } from '../../../../../../xpanse-api/generated';
+
+export default function ServicePolicyCreateResultStatus({
+    isError,
+    isSuccess,
+    error,
+    currentServicePolicy,
+}: {
+    isError: boolean;
+    isSuccess: boolean;
+    error: Error | null;
+    currentServicePolicy: ServicePolicy | undefined;
+}): React.JSX.Element {
+    if (isError) {
+        if (error instanceof ApiError && 'details' in error.body) {
+            const response: Response = error.body as Response;
+            return (
+                <div className={'submit-alert-tip'}>
+                    {' '}
+                    <Alert
+                        message={'Policy Process Status'}
+                        description={<ServicePolicySubmitResult msg={response.details.toString()} uuid={''} />}
+                        showIcon
+                        closable={true}
+                        type={'error'}
+                    />{' '}
+                </div>
+            );
+        } else {
+            return (
+                <div className={'submit-alert-tip'}>
+                    {' '}
+                    <Alert
+                        message={'Policy Process Status'}
```

Comment:
we need a relevant failure text here. in this case, we can write - Adding Policy Failed.

Applies to all failure alerts in this PR.